atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -224,8 +227,31 @@ def _render_mail(self, rebuild, success, auto_canceled, manual_canceled):
 
         url = self._get_logs_url()
 
+        repos = []
+        stored_data = self.workflow.exit_results.get(StoreMetadataInOSv3Plugin.key)
+        if not stored_data:
+            pass
+        elif 'annotations' not in stored_data:
+            pass
+
+        repo_data = json.loads(stored_data['annotations']['repositories'])
+
+        repos.extend(repo_data['unique'])
+        repos.extend(repo_data['primary'])
+
+        if not repos:
+            pass
+
+        image_name_obj = ImageName().parse(repos[0])
+        image_name = image_name_obj.namespace + '/' + image_name_obj.repo
```

Comment:
Not all images have a namespace. The `get_repo` method will handle that for you.